SU-HSDO/suhumsci
# Hsd8 546 node clone action

# READY FOR REVIEW

# Summary
- Clone node action that works with VBO module.
- VBO module enhances core actions module by allowing configurable actions.
- Changed core bulk actions on content list page for VBO actions

# Needed By (Date)
- asap

# Urgency
- Medium

# Steps to Test
1. checkout this branch
1. `composer update --prefer-source`
1. Sync a site `blt drupal:sync --site=[sitename]`
1. view the content list `/admin/content`
1. check a box on a node to clone
1. in the actions dropdown at the top, choose "Clone selected content" and click "apply to selected items"
1. in the following page, choose a number of how many times to clone the node and submit
1. validate the node was cloned with all field content in place.


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
